CHANGELOG.md

```diff
@@ -4,6 +4,262 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.14.0-RC1](https://github.com/go-gitea/gitea/releases/tag/v1.14.0) - 2021-03-09
+
+* SECURITY
+  * Add reverse proxy configuration support for remote IP address detection (#14959)
+  * Ensure validation occurs on clone addresses too (#14994)
+  * Fix several render issues highlighted during fuzzing (#14986)
+* BREAKING
+  * Restore Access.log following migration to Chi framework (Stops access logging of /api/internal routes) (#14475)
+  * Migrate from Macaron to Chi framework (#14293)
+  * Deprecate building for mips (#14174)
+  * Consolidate Logos and update README header (#14136)
+  * Inline manifest.json (#14038)
+  * Store repository data in data path if not previously set (#13991)
+  * Rename "gitea" png to "logo" (#13974)
+  * Standardise logging of failed authentication attempts in internal SSH (#13962)
+  * Add markdown support in organization description (#13549)
+  * Improve users management through the CLI (#6001) (#10492)
+* FEATURES
+  * Repository transfer has to be confirmed, if user can not create repo for new owner (#14792)
+  * Allow blocking some email domains from registering an account (#14667)
+  * Create a new issue based on reference to an issue comment (#14366)
+  * Add support to migrate from gogs (#14342)
+  * Add pager to the branches page (#14202)
+  * Minimal OpenID Connect implementation (#14139)
+  * Display current stopwatch in navbar (#14122)
+  * Display SVG files as images instead of text (#14101)
```

Comment:
```suggestion
  * Display SVG files as images instead of text (#14101)
  * Add [service.explore] settings to set access control of explore pages (#14094)
```

Reply:
-> already in enhancements